servers/src/main/java/tachyon/master/file/FileSystemMaster.java

```diff
@@ -1795,6 +1819,141 @@ private void setStateFromEntry(SetStateEntry entry) throws FileDoesNotExistExcep
   }
 
   /**
+   * Sets the acl of a file or directory. Only one of owner, group, or permission in the
```

Comment:
why "only one" instead of supporting being able to set at least one of owner, group, and permission?